src/parameterized_problem.cpp

```diff
@@ -685,3 +718,56 @@ LinElastDispLattice::LinElastDispLattice()
    general_vector_ptr[0] = NULL;
 
 }
+
+
+LinElastForceCantilever::LinElastForceCantilever()
+    : LinElastProblem()
+{
+   // pointer to static function.
+   bdr_type.SetSize(3);
+   battr.SetSize(3);
+   vector_bdr_ptr.SetSize(3);
+   for (size_t i = 0; i < vector_bdr_ptr.Size(); i++)
```

Comment:
Same as above for the readability: can we just not use the for loop here, since we write each case anyway?